cmd/crc/cmd/root.go

```diff
@@ -81,14 +81,15 @@ func setConfigDefaults() {
 	config.SetDefaults()
 }
 
-func exitIfMachineMissing(name string) {
+func checkIfMachineMissing(name string) error {
 	exists, err := machine.Exists(name)
 	if err != nil {
-		errors.ExitWithMessage(1, err.Error())
+		return err
 	}
 	if !exists {
-		errors.ExitWithMessage(1, "Machine '%s' does not exist. Use 'crc start' to create it.", name)
+		return fmt.Errorf("Machine '%s' does not exist. Use 'crc start' to create it", constants.DefaultName)
```

Comment:
@guillaumerose same here, in the CI it is failing with :( 

```
output did not match. Expected: 'Machine 'crc' does not exist. Use 'crc start' to create it.', Actual: 'Machine 'crc' does not exist. Use 'crc start' to create it'
```

Reply:
Fixed thanks